### DISCUSSION TITLE
How about support Crosscompiling officially ?

### DISCUSSION THREAD

**Discussion by wjhtinger** [eProsima/Fast-DDS#3652] [General]
DDS is used in embedded system anywhere, so I need help to Crosscompile the Fast-DDS.

**Comment by JLBuenoLopez** [comment]
Hi @wjhtinger 

The main issue about officially supporting crosscompiling is that there are a lot of targets. Currently you may find crosscompilation instructions for QNX in [Fast DDS documentation](https://fast-dds.docs.eprosima.com/en/latest/installation/sources/sources_qnx.html) (the QNX toolchain can be found in [Fast DDS repository](https://github.com/eProsima/Fast-DDS/blob/master/build_qnx/qnx.nto.toolchain.cmake)). This crosscompilation instructions were contributed by the community and that is the reason why QNX is currently a [Tier 3 supported platform](https://github.com/eProsima/Fast-DDS/blob/master/PLATFORM_SUPPORT.md#tier-3). You might want to contact [Fast DDS commercial support team](https://github.com/eProsima/Fast-DDS/tree/master#getting-help) for options about supporting other platforms you may be interested in.
